task-sdk/src/airflow/sdk/api/client.py

```diff
@@ -454,6 +455,18 @@ def get_state(self, dag_id: str, run_id: str) -> DagRunStateResponse:
         return DagRunStateResponse.model_validate_json(resp.read())
 
 
+class TaskRescheduleOperations:
```

Comment:
I think this should be added to the existing TI Operations rather than create a whole new one

Reply:
I considered too, but since I created a new API routes for Reschedule I went with this.. No strong reason either for that too.. So if you have a strong opinion have to change that